services/auth.js

```diff
@@ -31,8 +36,20 @@ const authService = {
 
     const hashedPassword = await hashPassword(password)
     const activationLink = uuidv4()
+    const foundRole = await Role.findOne({ value: role }).exec()
+
+    if (!foundRole) {
+      throw createError(404, ROLE_NOT_SUPPORTED)
```

Comment:
maybe better move status code to consts or get from from consts if it already exists

Reply:
following the pattern that was created before me, was told not to change it